tests/buildnothing/Makefile.BSD

```diff
@@ -0,0 +1,4 @@
+test:
+	echo "Exciting Makefile, huh?  Probably run a script here?"
```

Comment:
This should be `echo "SUCCESS"`, I think?

Anything which is hooked into the tests should behave like a test...

Reply:
well, it's not being run by `make test` (which is now `tests/test_libcperciva.sh`, so I figured it didn't matter.  But ok.